.github/workflows/config/lint-config.yml

```diff
@@ -108,7 +108,7 @@ linters-settings:
     # max line length, lines longer will be reported. Default is 120.
     # '\t' is counted as 1 character by default, and can be changed with the
     # tab-width option
-    line-length: 120
+    line-length: 180
```

Comment:
I think this change should be part of the first commit itself, where these changes were introduced.

Reply:
done.